nilearn/connectome/connectivity_matrices.py

```diff
@@ -390,4 +521,69 @@ def transform(self, X):
                                  '"covariance" and "precision", got kind '
                                  '"{}"'.format(self.kind))
 
-        return np.array(connectivities)
+        connectivities = np.array(connectivities)
+        if self.kind != 'tangent':
+            self.mean_ = connectivities.mean(axis=0)
+
+        if self.vectorize:
+            connectivities = sym_matrix_to_vec(
+                connectivities, discard_diagonal=self.discard_diagonal)
+
+        return connectivities
+
+    def _check_fitted(self):
+        if not hasattr(self, "cov_estimator_"):
+            raise ValueError('It seems that {0} has not been fitted. '
+                             'You must call fit() before calling '
+                             'transform().'.format(self.__class__.__name__)
+                             )
+
+    def inverse_transform(self, connectivities, diagonal=None):
+        """Returns connectivity matrices from vectorized or not connectivities.
```

Comment:
I cannot understand the end of the sentence: "from vectorized or not connectivities." Do you mean "from connectivities, vectorized or not"?